Remove possible contradictions between the GSG and the Cypher manual (#287)
neo4j/docs-getting-started: modules/ROOT/pages/cypher-intro/results.adoc

+--------+
----

If you want to skip `null` values, use the function `count(variable)`. 

> For aggregating only unique values use the `DISTINCT` operator, for example: `count(DISTINCT role)`.

Reviewer: Should these two not have full example queries?

Author: IMO, you're right. It's good to illustrate them with code examples. I'll make a note for myself for future

+--------------------------------------------------------------------------------------------------------------+
----

> Therefore, there are three different ways to use function `count()`:

Reviewer: ```suggestion
There are three different ways to use the `count()` function:
```

> . To count results with `count({empty}*)`, which simply returns the number of matching rows.

Reviewer: ```suggestion
. `count({empty}*)`: counts results and returns the number of matching rows.
```

Is `{empty}` clear syntax here? it is not used in Cypher Manual (AFAIK), and Im not really sure what it means?

> . To use `count(n)` to count the number of occurences of `n` and to exclude `null` values.

Reviewer: ```suggestion
. `count(n)`: count the number of occurrences of `n` (excludes `null` values).
```

You can specify nodes, relationships, or properties within the parentheses for Cypher to count.

> . To use the `DISTINCT` operator -- `count(DISTINCT variable)` -- to remove duplicates from the results.

Reviewer: ```suggestion
. The `DISTINCT` operator -- `count(DISTINCT variable)` -- removes duplicates from the results.
```

> In the dataset <<people-technologies-companies, _People, technologies, and companies_>>, some of the `Person` nodes have a Twitter handle, but others do not.

Reviewer: Is this link useful here? If you click it, it is pretty disruptive to the reading experience. Maybe just, "In the dataset,..." - as it doesnt change on the page?

Author: I think, it's good to have this link here as this piece of content has been clued from two diff. chapters and we use here two datasets: Movie and People, companies.... In the future, I'd like to rewrite all examples to use only one dataset, but in the meantime, it's good to remind users what dataset is used.
